jobs/v3/howto/location_based_search_sample.go

```diff
@@ -82,9 +82,9 @@ func basicLocationSearch(w io.Writer, projectID, companyName, location string, d
 	return resp, nil
 }
 
-// [END basic_location_search]
+// [END job_basic_location_search]
 
-// [START city_location_search]
+// [START job_city_location_search]
```

Comment:
you need to leave city_location_search here until the new region tag has been merged.

Reply:
Thank you for catching this. I reverted this tag for now.

Reply:
what you can do is add the new one, just don't delete the old one and then after CL merges you can delete the old one.